modules/layers/src/text-layer/text-layer.ts

```diff
@@ -365,19 +366,38 @@ export default class TextLayer<DataT = any, ExtraPropsT extends {} = {}> extends
     });
   }
 
+  private transformParagraph(
```

Comment:
Short JSDoc on each method would be nice. For instance, hard to know what `transform` means when just reading fragments here in the PR, a brief comment could make that clear.

Reply:
Noted that you didn't add JSDoc. I can certainly stop suggesting it, but I do think every method and type should have at least one line of JSDoc (no need to doc params just a `/** ... */` line explaining the purpose).

Reply:
You are looking at an older commit.